doc/user/bgp.rst

```diff
@@ -3470,6 +3470,15 @@ starting the daemon and the configuration gets saved, the option will persist
 unless removed from the configuration with the negating command prior to the
 configuration write operation.
 
+.. index:: [no] bgp send-extra-data zebra
+.. clicmd:: [no] bgp send-extra-data zebra
+
+   This Command turns off the ability of BGP to send extra data to zebra.
+In this case it's the AS-Path being used for the path.  The default behavior
+in BGP is to send this data and to turn it off enter the no form of the command.
```

Comment:
how do we select which data to send to zebra ?
today, it is as path information, tomorrow it may be route distinguisher.

will RD be possible ?

Reply:
it's certainly possible.  Someone needs to code it up.  Feel free